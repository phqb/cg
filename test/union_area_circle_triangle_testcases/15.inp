6 0 4
2.4166666667 -100
100.0000000000 99.5
2.4166666667 1.9166666667