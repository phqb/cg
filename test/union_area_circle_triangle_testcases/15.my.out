49.1284597151